packages/rancher-cis-benchmark/charts/values.yaml

```
@@ -45,7 +45,7 @@ global:
     clusterName: ""
   kubectl:
     repository: rancher/kubectl
-    tag: v1.28.7
+    tag: v1.29.2
```

```suggestion
    tag: v1.29.2
```
Not 100% sure about this version of kubectl, based on [kubelet version skew policy](https://kubernetes.io/releases/version-skew-policy/#kubelet), if we have to cover up to k8s 1.31 we might need kubectl 1.30 at least, since it covers version-1 | version | version+1.

---

the minimum version supported by this chart is supposed to be 1.28. So based on that, it seems 1.29.x is a better fit at least until 1.31 support is added.